tests/utils/test_qtutils.py

```diff
@@ -20,107 +20,73 @@
 """Tests for qutebrowser.utils.qtutils."""
 
 import sys
-import unittest
+
+import pytest
 
 from qutebrowser import qutebrowser
 from qutebrowser.utils import qtutils
+import overflow_test_cases
```

Comment:
Does this actually work? I mean there is no ``__init__.py`` or path manipulation :confused: